### DISCUSSION TITLE
【SOUNDRAW】AI 音樂生成器

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#52] [科技新知]
## 簡介
SOUNDRAW 是一個 AI 音樂生成器，允許所有創作者自由定制獨特、免版稅音樂的長度、組成和樂器
實際聽了DEMO音樂後，覺得還可以，因為沒有讓我感到很精豔
但是，用AI來做音樂生成，並且製作成web app來提供商品服務，這件事情很棒。
如果將來同學們的專題是做出AI商業服務網站，這種作品集對將來就業或升學一定很有說服力阿~

## 參考網站：
1. 官網：https://soundraw.io/
2. 某一個販售網站：https://appsumo.com/products/soundraw/

**Comment by HSU-YU-JEN** [comment]
這個好酷~我從沒有想到過AI可以學習音樂

**Comment by enoch-lightning** [comment]
蝦皮購物、露天購物與亞瑪遜等購物網站，可以用購物車存放要買的東西，並計算價錢。因為不是人親自去做的，而是使用電腦計算，這樣就算是「人工智慧=artificial intelligence=AI」嗎？
